### PR TITLE
[WIP]AMQ Streams Deployment file

### PR DESCRIPTION
**Pull request by ksandha**

### CODE REVIEW

File: ocs_ci/templates/openshift-infra/amq/examples/templates/cluster-operator/connect-s2i-template.yaml [hunk 1]
```diff
@@ -0,0 +1,104 @@
+apiVersion: v1
```
Comment (vasukulkarni) [inline ocs_ci/templates/openshift-infra/amq/examples/templates/cluster-operator/connect-s2i-template.yaml]: Are these files from openshift-infra? Can it be cloned and only things that we change can be in our yaml directory?

Comment (vasukulkarni) [reply ocs_ci/templates/openshift-infra/amq/examples/templates/cluster-operator/connect-s2i-template.yaml]: Also the directory name for templates should be templates/workloads/amq

File: ocs_ci/utility/workloads/deploy_amq_stream.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+)
+
+logger = logging.getLogger(__name__)
+
+@pytest.mark.usefixtures(
+    create_cephfs_secret.__name__,
+    create_cephfs_storageclass.__name__,
+)
+
+def create_namespace(yaml_file):
```
Comment (vasukulkarni) [inline ocs_ci/utility/workloads/deploy_amq_stream.py]: we have this function in OCP class (create new project)

**Comment by vasukulkarni** [comment]
test file needed along with utility functions.

File: ocs_ci/templates/openshift-infra/amq/examples/templates/cluster-operator/connect-s2i-template.yaml (hunk 1 above)
Comment (ksandha) [reply ocs_ci/templates/openshift-infra/amq/examples/templates/cluster-operator/connect-s2i-template.yaml]: 1) The files are gather from a downloadable link which needs authentication. The link is attached [here](https://access.redhat.com/jbossnetwork/restricted/listSoftware.html?downloadType=distributions&product=jboss.amq.streams)
2) I have kept the path similar to the .zip file which contains the yaml files. With this we will be able to make out which yaml files are linked to which directory.

**Comment by ebenahar** [comment]
Closing old PRs. Please re-open if needed

**Pull request closed without merging**